.github/workflows/checks.yml

```diff
@@ -91,11 +91,11 @@ jobs:
           DEVNET_PATH: ${{ github.workspace }}/starknet-devnet-rs/target/release/starknet-devnet
           NETWORK_TEST_MODE: "disabled"
           NETWORK_TEST_NETWORK_NAME: "TESTNET"
-          TESTNET_RPC_URL: ${{ secrets.TESTNET_RPC_URL }}
-          TESTNET_ACCOUNT_ADDRESS: ${{ secrets.TESTNET_ACCOUNT_ADDRESS }}
-          TESTNET_PRIVATE_KEY: ${{ secrets.TESTNET_PRIVATE_KEY }}
-          TESTNET_CONST_NONCE_ACCOUNT_ADDRESS: ${{ secrets.TESTNET_CONST_NONCE_ACCOUNT_ADDRESS }}
-          TESTNET_CONST_NONCE_PRIVATE_KEY: ${{ secrets.TESTNET_CONST_NONCE_PRIVATE_KEY }}
+          GOERLI_TESTNET_RPC_URL: ${{ secrets.TESTNET_RPC_URL }}
+          GOERLI_TESTNET_ACCOUNT_ADDRESS: ${{ secrets.TESTNET_ACCOUNT_ADDRESS }}
+          GOERLI_TESTNET_PRIVATE_KEY: ${{ secrets.TESTNET_PRIVATE_KEY }}
+          GOERLI_TESTNET_CONST_NONCE_ACCOUNT_ADDRESS: ${{ secrets.TESTNET_CONST_NONCE_ACCOUNT_ADDRESS }}
+          GOERLI_TESTNET_CONST_NONCE_PRIVATE_KEY: ${{ secrets.TESTNET_CONST_NONCE_PRIVATE_KEY }}
```

Comment:
This can be removed altogether as lately devnet delivers 💪🏻 
But can also be kept to make setup easier, should there be a need for it.

Reply:
I've quickly reviewed the workflow logs and it seems that all tests are currently skipped. Is this the expected outcome?

Reply:
Network tests were meant to be run locally when devnet was unreliable or lacked updates. They were never really meant to run on CI; it was a temporary workaround.

I only turned some of them (non-gas ones) on CI when there was a huge delay in devnet release, and later disabled them due to network issues and a lack of need to use those.

That's why my initial comment suggests that the network test config can probably be removed from the workflow until it's needed. But there's also a reason to keep it.

Reply:
Thanks for the clarification. As discussed offline the logs lack information about successful tests - they only contain details about failed and skipped ones. 

Imo it would be beneficial to have some basic tests on integration/testnet, but that's to discuss with the rest of the team.

I am fine with either leaving or removing the testnet environment variables. Although, my intuition is that they might be useful at some point :smiley: